<launch>
	<arg name="video_topic" default="/camera/rgb/image_color"/>
	<arg name="depth_topic" default="/camera/depth/image_raw"/>
	<arg name="frame_width" default="640"/>
	<arg name="frame_height" default="480"/>
	<arg name="video_codec" default="MJPG"/>
	<arg name="file_loc" default="/home/RGBout.avi"/>
	<node name="compress_node" pkg="final_project" type="compress_node" respawn ="false" output ="screen">
		<param name="video" value="$(arg video_topic)" type="string"/>
		<param name="depth" value="$(arg depth_topic)" type="string"/>
		<param name="width" value="$(arg frame_width)" type="int"/>
		<param name="height" value="$(arg frame_height)" type="int"/>
		<param name="codec" value="$(arg video_codec)" type="str"/>
		<param name="file" value="$(arg file_loc)" type="string"/>
	</node>
</launch>
